test/unit/mock-tooling.ts

```diff
@@ -19,4 +30,26 @@ describe('mock-tooling', () => {
       return expect(() => mockTooling.mockTokeninfoEndpoint({ url: ''} as MockOptions)).to.throw(Error);
     });
   });
+
+  describe('mockAccessTokenEndpointWithErrorResponse', () => {
+    it('should throw error, if url is empty', () => {
+      return expect(() => mockTooling.mockAccessTokenEndpointWithErrorResponse({ url: ''} as MockOptions, HttpStatus.BAD_REQUEST)).to.throw(Error);
```

Comment:
`as MockOptions` should not be necessary. I guess its c&p from line 24 where for some reason a prop `auth: ''` is defined, I think that can be removed and then we do not need the `as` there anymore.